src/static/js/pad.js

```diff
@@ -669,6 +678,12 @@ const pad = {
       $('#options-stickychat').prop('checked', false); // set the checkbox for off
     }
   },
+  determineFocus: (asNowConnectedFeedback) => {
+    const focus = padcookie.getPref('focusOnEditor');
+    if (focus) {
+      padeditor.ace.focus();
```

Comment:
Is this necessary? `focus()` is already called after pad init.

Reply:
is pad init called upon reconnect?

Reply:
No, but is there a reason `focus()` needs to be called on reconnect? Seems to me that it would be better to not call it.

Reply:
I'd of thought so but I'm not passionate about it.  My goal of using the additional reconnect was to keep the logic the same as other reconnect logic but I'm easy either way :)

Reply:
I pushed a change so that it doesn't re-steal focus on reconnect.